.github/workflows/scripts/run_affected_benchmarks

```diff
@@ -145,6 +145,12 @@ main() {
 	echo "Finding C benchmark files in ${directories}..."
 	c_bench_files=$(find "${directories}" -maxdepth 5 \( -wholename '**/benchmark/c/benchmark*.c' -or -wholename '**/benchmark/c/**/benchmark*.c' \) -exec realpath {} \; | grep -v '/fixtures/' | sort -u | tr '\n' ' ') || true
 
+	# If benchmarks requiring Cephes are found, install the Cephes library:
+	c_cephes_makefiles=$(find "${directories}" -maxdepth 5 -wholename '**/benchmark/c/**/Makefile' -exec grep -l 'CEPHES' {} + || true)
```

Comment:
Targeting the `CEPHES` environment variable is a bit brittle (this could always change), but I don't see a better alternative atm.